corehq/apps/app_manager/views.py

```diff
@@ -1232,7 +1232,8 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
                             unique_form_id=form.unique_id)
 
     vellum_plugins = ["modeliteration"]
-    if toggles.VELLUM_ITEMSETS.enabled(domain):
+    if (toggles.VELLUM_ITEMSETS.enabled(domain) and request.subscription):
+        # request.subscription checks it's not a community subscription
```

Comment:
@emord You want to follow this pattern: https://github.com/dimagi/commcare-hq/pull/5750

Sometimes community subscriptions exist explicitly, other times the subscription is just ```None```